airflow/migrations/versions/0109_424117c37d18_add_dagconfiguratioerror_model.py

```diff
@@ -0,0 +1,75 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Add DagConfiguratioError model
+
+Revision ID: 424117c37d18
+Revises: b1b348e02d07
+Create Date: 2022-04-27 15:57:36.736743
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+from sqlalchemy import Column, Integer
+
+from airflow.migrations.db_types import StringID
+
+revision = '424117c37d18'
+down_revision = 'b1b348e02d07'
+branch_labels = None
+depends_on = None
+airflow_version = '2.4.0'
+
+
+def upgrade():
+    """Apply Add DagConfiguratioError model"""
+    op.create_table(
+        'dag_configuration_warning',
```

Comment:
I wonder if we should just call this `dag_warning`; I recall I wanted this for something else on parse-time that isn’t quite `ImportError` (don’t remember exactly what).

Reply:
I like dag warning 👍